### PR TITLE
Fix objects

### PR DESCRIPTION
**Pull request by glsubri**

### CODE REVIEW

File: app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionUtils.java [hunk 1]
```diff
@@ -14,13 +16,41 @@
...
+
+        handler.postDelayed(new Runnable(){
+            public void run(){
+                Rockin.api()
+                        .getQuestions(poll.getIdModerator(), poll.getIdPoll(), token)
+                        .enqueue(callbackQuestions);
+                handler.postDelayed(this, DELAY_IN_MILLIS);
+            }
+        }, DELAY_IN_MILLIS);
+    }
```
Comment (alexandrepiveteau) [inline app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionUtils.java:51]: A bit ugly tbh. I think you should add an issue so we can improve this and fix it later.

**Review by alexandrepiveteau — APPROVED** [review]
Okay for me. Please add two issues to :

- Figure out a better way to fix navigation.
- Better manage the questions state.

File: app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionUtils.java (hunk 1 above)
Comment (glsubri) [reply app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionUtils.java:51]: I think that this is the android way for small intervals. Have you seen anything else done ?

https://developer.android.com/reference/android/os/Handler

Comment (alexandrepiveteau) [reply app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionUtils.java:51]: It's more the fact that it's static that is not awesome :D

**Pull request merged**